app/assets/javascripts/oxalis/view/user_scripts_modal.js

```diff
@@ -43,7 +78,26 @@ class UserScriptsModalView extends React.PureComponent {
         onOk={this.handleClick}
         onCancel={this.props.onClose}
       >
-        <Input type="textarea" rows={4} onChange={this.handleCodeChange} value={this.state.code} />
+        <Spin spinning={this.state.isLoading}>
+          <Select
+            value={this.state.selectedScript}
+            style={{ width: 200, "margin-bottom": 10 }}
```

Comment:
You get a big fat warning for React this way. React expects camelCased CSS keywords.